File: 27-rotate-by-one.py
# Subject: Given an array, cyclically rotate the array clockwise by one. 

# Example:
# Input:  arr[] = {1, 2, 3, 4, 5}
# Output: arr[] = {5, 1, 2, 3, 4}

# Reference => https://www.geeksforgeeks.org/c-program-cyclically-rotate-array-one/


def rotateListByOne(list):
    rotatedList = list[:] = list[-1:] + list[:-1]
    print("{}".format(rotatedList))

rotateListByOne([1, 2, 3, 4, 5])

File: 27-rotate-by-one.js
// Subject: Given an array, cyclically rotate the array clockwise by one. 

// Example:
// Input:  arr[] = {1, 2, 3, 4, 5}
// Output: arr[] = {5, 1, 2, 3, 4}

// EX 1:
function rotateByOne(arr=[]){
    let arrPop = arr.pop();
    arr.unshift(arrPop);
    return arr;
    
}

rotateByOne([1, 2, 3, 4, 5]);

